### PR TITLE
[bitnami/spring-cloud-dataflow] Support for customizing standard labels

### PR DESCRIPTION
**Pull request by juan131**
### Description of the change

This PR follows up #18154 ensuring this chart is adapted to use new helpers' format, hence adding support for customizing standard labels.

### Benefits

User can customize standard labels.

### Possible drawbacks

None

### Applicable issues

N/A

### Additional information

#18154 must be merged **before** this PR

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by juan131** [comment]
Blocking until we bumped the major version on a different PR to use new Kafka major.

**Comment by juan131** [comment]
Unblocking PR

File: bitnami/spring-cloud-dataflow/Chart.yaml [hunk 1]
```diff
@@ -53,4 +53,4 @@ maintainers:
 name: spring-cloud-dataflow
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/spring-cloud-dataflow
-version: 23.0.0
+version: 22.1.0
```
Comment (joancafom) [inline bitnami/spring-cloud-dataflow/Chart.yaml:56]: Bump the minor version properly

```suggestion
version: 23.1.0
```

File: bitnami/spring-cloud-dataflow/templates/server/tls-secret.yaml [hunk 2]
```diff
@@ -33,10 +30,6 @@ kind: Secret
 metadata:
   name: {{ $secretName }}
   namespace: {{ .Release.Namespace }}
-  labels: {{- include "common.labels.standard" . | nindent 4 }}
-    {{- if .Values.commonLabels }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
-    {{- end }}
```
Comment (joancafom) [inline bitnami/spring-cloud-dataflow/templates/server/tls-secret.yaml:39]: You completely removed the labels here

Comment (juan131) [reply bitnami/spring-cloud-dataflow/templates/server/tls-secret.yaml:39]: fixed at https://github.com/bitnami/charts/pull/18749/commits/edc3ac67669f94c68712b8fe91c047ce6979a4e2

File: bitnami/spring-cloud-dataflow/Chart.yaml (hunk 1 above)
Comment (juan131) [reply bitnami/spring-cloud-dataflow/Chart.yaml]: fixed at https://github.com/bitnami/charts/pull/18749/commits/edc3ac67669f94c68712b8fe91c047ce6979a4e2

**Review by joancafom — APPROVED** [review]

**Pull request merged**